.gitignore

```diff
@@ -4,6 +4,8 @@
 /.nyc_output
 /node_modules
 /out
+/asm-docs*
```

Comment:
I'm intrigued why this needed to be added - what creates this?

Reply:
This is created by the python script.  It extracts the opcode archives into those folders for parsing.  I'm a little surprised it wasn't already in there.

Reply:
I don't think we're still using the script, nor have I ever ran it myself, so that might be the reason why it wasn't there.

Reply:
we do still use the script, just not super frequently (I myself have run it twice).  However, this path is not hard-coded. I usually run the script with flags which place the temp files out-of-tree.

[Of all the commits](https://github.com/compiler-explorer/compiler-explorer/commits/main/lib/handlers/asm-docs.js), only the most recent has applied a manual tweak.

If we leave this in the gitignore can we please add a comment indicating why it is relevant?

Reply:
+1 Either remove or put a comment please! thanks!